public/app/percona/backup/components/Status/Status.styles.ts

```diff
@@ -13,14 +13,20 @@ export const getStyles = stylesFactory((theme: GrafanaTheme, status: BackupStatu
   ];
   const isSuccess = successfulStates.includes(status);
   const isError = errorStates.includes(status);
+  const result = {
+    ellipsisContainer: css`
+      display: table;
+      width: 20px;
+      margin: 0 auto;
+    `,
+    status: '',
+  };
 
   if (isSuccess || isError) {
```

Comment:
I think you could benefit from conditional styles with the `cx` function so the styles file would be just the styles and  the conditions/logic would be on the component.

Like here: https://github.com/percona-platform/grafana/blob/7dffd14498dbd9900ab5f596c71364be6786a8a3/public/app/percona/dbaas/components/DBCluster/DBClusterStatus/DBClusterStatus.tsx#L20

Just a suggestion anyway.

Reply:
Thanks, way cleaner !